packages/language-server/test/completion-items-classes-spec.ts

```diff
@@ -332,6 +333,50 @@ describe("the UI5 language assistant Code Completion Services - classes", () =>
       expect(getTextInRange(xmlSnippet, suggestion.textEdit?.range)).to.equal(
         ""
       );
+      // Check the namespace is added at the correct position
+      expect(suggestion.additionalTextEdits, "additionalTextEdits").to.be.empty;
+      const tagName = getTagName(suggestion.textEdit);
+      expectExists(tagName, "tag name in suggestion");
+      const ns = tagName.split(":")[0];
+      const attributes = getAttributes(suggestion.textEdit);
+      expect(
+        attributes,
+        `attributes of ${suggestion.textEdit?.newText}`
+      ).to.have.lengthOf(2); // Namespace and tab stop
+      expect(
+        attributes[0],
+        `attributes[0] of ${suggestion.textEdit?.newText}`
+      ).to.match(
+        new RegExp(
+          `^xmlns:${ns}=`
+        ) /*, `attribute should start with xmlns:${ns}=`*/
+      );
+    });
+  });
+
+  it("will return valid class suggestions for empty tag with closing bracket", () => {
```

Comment:
for future upgrades: snapshots testing may make writing these tests much simpler.

Reply:
These tests look at all the returned results, not a specific case. I'm not sure I would want a test that lists all possible classes in the root.